### PR TITLE
[DA-3639] Remote ID Verification - Update PS fields to display UNSET in API 

### PR DESCRIPTION
**Pull request by ayazamlani**
## Description of changes/additions
This change displays UNSET in the API when NULL in the participant_summary table for certain fields related to Remote ID Verification.

## Tests
- [] unit tests

[DA-3639]: https://precisionmedicineinitiative.atlassian.net/browse/DA-3639?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by kenny-skaggs — APPROVED** [review]

File: rdr_service/dao/questionnaire_response_dao.py [hunk 1]
```diff
@@ -948,10 +948,10 @@ def _update_participant_summary(
                 participant_summary.idVerificationOrigin = IdVerificationOriginType.REMOTE
                 if not participant_summary.firstIdVerifiedOn:
                     participant_summary.firstIdVerifiedOn = datetime.utcfromtimestamp(remote_id_info['verified_on'])
```
Comment (pegbertsch) [inline rdr_service/dao/questionnaire_response_dao.py:955]: Do you have to confirm first now that the `verified_on` field has a value before you try to use it as a timestamp, now that it's not checked on line 942?    

If there's a chance the value could ever be missing (even if that's not expected), I think line 950 would throw an  exception in the `utcfromtimestamp` call.   Also, isn't `firstIdVerifiedOn` a Date field (vs. Datetime) in the model?

File: tests/api_tests/test_participant_summary_api.py [hunk 2]
```diff
@@ -4452,49 +4457,78 @@ def test_updated_since_parameter(self):
...
             questionnaire_id,
             participant_id
         )
         self.send_post("Participant/%s/QuestionnaireResponse" % participant_id, resource)
-        # Get request from API to assert information is accurate
+        # Get request from API to assert True Response information is accurate
         summary = self.send_get("Participant/%s/Summary" % participant_id)
         self.assertEqual(summary['remoteIdVerificationOrigin'], 'example')
-        self.assertEqual(summary['remoteIdVerificationStatus'], True)
+        self.assertEqual(summary['remoteIdVerificationStatus'], 'True')
```
Comment (pegbertsch) [inline tests/api_tests/test_participant_summary_api.py:4496]: Not clear why this (and line 4503, 4516...) is being tested as the string 'True', as opposed to `assertTrue` ?  These are Boolean columns in the model?

File: rdr_service/dao/participant_summary_dao.py [hunk 3]
```diff
@@ -1241,6 +1242,19 @@ def build_default_obj_response(self, obj: ParticipantSummary):
...
+        # Format True False None Responses to default to UNSET when none
+        field_names = [
+            'remoteIdVerificationStatus',
+            'everIdVerified'
+        ]
+        for field_name in field_names:
+            if result.get(field_name) is None:
+                result[field_name] = UNSET
+            elif result[field_name]:
+                result[field_name] = 'True'
```
Comment (pegbertsch) [inline rdr_service/dao/participant_summary_dao.py:1254]: These need Boolean `True` or `False` , not strings...correct?

File: tests/api_tests/test_participant_summary_api.py (hunk 2 above)
Comment (ayazamlani) [reply tests/api_tests/test_participant_summary_api.py:4496]: Yes, these are booleans in the db, but returning as strings in the API to be consistent. Recommended by @kenny-skaggs [here](https://pmi-engteam.slack.com/archives/C03TQ7D42RL/p1688764363292639).

File: rdr_service/dao/questionnaire_response_dao.py (hunk 1 above)
Comment (ayazamlani) [reply rdr_service/dao/questionnaire_response_dao.py:955]: thanks @pegbertsch , I have added the code for the verified_on part of this review. 

yes, firstIdVerifiedOn is a date field in the model. This is using the datetime.datetime python library to get the utctimestamp, and then saving as a Date in the db.

File: rdr_service/dao/participant_summary_dao.py (hunk 3 above)
Comment (ayazamlani) [reply rdr_service/dao/participant_summary_dao.py:1254]: [same](https://github.com/all-of-us/raw-data-repository/pull/3553#discussion_r1279613335)

**Review by pegbertsch — APPROVED** [review]

**Pull request merged**